pyproject.toml

```diff
@@ -90,7 +97,6 @@ jinja2 = "^3.1.2"
 LinkChecker = "^10.1.0"
 kaggle = "^1.5.12"
 tf2onnx = "^1.10.1"
-types-requests = "^2.28.0"
```

Comment:
why removing this ?

Reply:
Because we removed requests from our requirements so it seemed odd to have it here

Reply:
gotcha !